daily-challenges/2024-11/05/ththao27.spec.ts

```
@@ -0,0 +1,25 @@
+function calculateTreasure(sequences: string[]): number[] {
+    return sequences.map((sequence) => {
+        const match = sequence.match(/(-?\d+)\D+(-?\d+)/);
+        let sum = 0;
+        if (match) {
+            const start = parseInt(match[1]);
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)